src/Runners/PHPUnit/Options.php

```diff
@@ -739,6 +756,12 @@ public static function setInputDefinition(InputDefinition $inputDefinition): voi
                 InputOption::VALUE_OPTIONAL,
                 'Use a specific random seed <N> for random order'
             ),
+            new InputOption(
+                'repeat',
+                'r',
```

Comment:
I prefer no shortcuts, like PHPUnit:
```suggestion
                null,
```

Reply:
Ooops, okay.